Backend/app/provider-platform/dynamic-offer-driver-app/Main/spec/Storage/Merchant.yaml

```diff
@@ -652,6 +652,21 @@ TransporterConfig:
       overlay: Int
       alert: Int
       derive': Generic, Show, ToJSON, FromJSON, Read
+    ArrivalTimeBufferOfVehicle:
```

Comment:
Show we make all these as Maybe Seconds, so that we only do this showing estimated duration Computation for deliverybike as of now to keep Google cost under control.